docs/community.html

```diff
@@ -0,0 +1,266 @@
+      <br>
+      <div>
+        <p class="ov-community-paragraph">
+         Reporting bugs is one of the most valuable ways you can contribute! Ideas for new features are also very welcome. Report bugs and RFEs using the following issue trackers:
+        </p>    
+        <ul style="list-style-type:disc;" class="ov-community-list">
+          <li>
+            <a href="https://bugzilla.redhat.com/enter_bug.cgi?classification=oVirt">Bugzilla - bugs and RFEs in oVirt Engine and oVirt Node</a>
+          </li>
+          <li>
+            <a href="https://github.com/oVirt/ovirt-web-ui/issues">VM Portal GitHub - bugs and RFEs in VM Portal</a>
+          </li>
+          <li>
+            <a href="https://github.com/oVirt/ovirt-site/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation">oVirt Documentation GitHub - Documentation issues</a>
+          </li>
+          <li>
+            Filing a security issue following <a href="https://ovirt.org/develop/security/security.html">this reporting procedure</a>
+          </li>
+        </ul>
+      </div>  
+    </section>
+
+    <section class="ov-site-section">
+      <h2 class="ov-heading ov-heading--lg">Participate in the oVirt Infrastructure</h2>
+      <br>
+      <div>
+        <p class="ov-community-paragraph">
+        Our project infrastructure can always benefit from extra people, hardware and network bandwidth. First of all, consider <a href="https://ovirt.org/develop/infra/repository-mirrors.html">becoming a public mirror</a> of oVirt repositories. You can also <a href="https://ovirt.org/develop/infra/repository-mirrors.html">become an infrastructure team member</a> or <a href="https://ovirt.org/community/get-involved/donate-hardware.html">donate hardware</a> to the project to improve our capacity and redundancy.
+        </p>    
+      </div>  
+    </section>
+
+    <section class="ov-site-section">
+      <h2 class="ov-heading ov-heading--lg">Supporters, Sponsors, and Providers</h2>
+      <br>
+      <div>
+        <p class="ov-community-paragraph">
+         oVirt has a broad community of <a href="https://ovirt.org/community/user-stories/users-and-providers.html">supporters, sponsors, and providers</a>.
+        </p>    
+      </div>  
+    </section>
+
+    <section class="ov-site-section">
+      <h2 class="ov-heading ov-heading--lg">Social Media</h2>
+      <br>
+      <div> 
+        <p>
+          Follow us on: 
+        </p>
+        <ul style="list-style-type:disc; "class="ov-community-list">
+          <li>
+            <a href="https://twitter.com/ovirt">Twitter</a>
+          </li>
+          <li>
+            <a href="https://www.facebook.com/groups/ovirt.openvirtualization/">Facebook</a>
+          </li>
+          <li>
+           <a href="https://www.youtube.com/user/ovirtproject">YouTube</a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="ov-site-section">
```

Comment:
I wrote the `ov-site-section` to be more of a entire content wrapper so you might want to create something unique here. You could probably also tackle this without wrapping everything in sections but just have one big wrapper for all this content.

Reply:
I ended up putting all of the subsections into a div and added `ov-site-section` as the class for the overall wrapper and then added a specific class for the margins of the subsections.